app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
```

Comment:
- [ ] One last thing is, that I could not initialize the app and access the login and signup pages, Once you set the devise Gem properly, Please make sure the views in the devise for the registration and login are modified to match the wireframes and their styling.